service/resource/cloudformation/adapter.go

```diff
@@ -0,0 +1,81 @@
+package cloudformation
+
+import (
+	"github.com/giantswarm/awstpr"
+	"github.com/giantswarm/microerror"
+
+	awsutil "github.com/giantswarm/aws-operator/client/aws"
+)
+
+type hydrater func(awstpr.CustomObject, awsutil.Clients) error
+
+type adapter struct {
+	ASGType string
+
+	lauchConfigAdapter
+	autoScalingGroupAdapter
+}
+
+type lauchConfigAdapter struct {
+	ImageID                  string
+	SecurityGroupID          string
+	InstanceType             string
+	IAMInstanceProfileName   string
+	BlockDeviceMappings      []BlockDeviceMapping
+	AssociatePublicIPAddress bool
+	SmallCloudConfig         string
+}
```

Comment:
Nitpick but I think having these ordered alphabetically would be good. We found that useful in the old style AWS resources.

Reply:
Done